test/performance/README.md

```diff
@@ -49,6 +49,58 @@ if err != nil {
 }
 ```
 
+## Zipkin trace
+
+Zipkin tracing can be enabled if needed by the performance test during setup.
+
+```
+perfClients, err := Setup(t, EnableZipkinTracing)
+```
+
+Once it is setup, all requests made by the `SpoofingClient` can be logged and saved to a trace file using the
```

Comment:
```suggestion
Once enabled, all requests made by the `SpoofingClient` will have an additional trace header. This can be used to get the entire request trace and store it in a trace file in the artifacts directory using the
```